diff --git a/ophyd/sim.py b/ophyd/sim.py
index 4ff44dc..4cfa476 100644
--- a/ophyd/sim.py
+++ b/ophyd/sim.py
@@ -12,6 +12,7 @@ import weakref
 import uuid
 import copy
 import logging
+from functools import partial
 
 from .signal import Signal, EpicsSignal, EpicsSignalRO
 from .areadetector.base import EpicsSignalWithRBV
@@ -649,7 +650,8 @@ class SynSignalWithRegistry(SynSignal):
     """
 
     def __init__(self, *args, reg=DO_NOT_USE, save_path=None,
-                 save_func=np.save, save_spec='NPY_SEQ', save_ext='npy',
+                 save_func=partial(np.save, allow_pickle=False),
+                 save_spec='NPY_SEQ', save_ext='npy',
                  **kwargs):
         super().__init__(*args, **kwargs)
         self.save_func = save_func
@@ -768,7 +770,8 @@ class NumpySeqHandler:
         self._name = os.path.join(root, filename)
 
     def __call__(self, index):
-        return np.load('{}_{}.npy'.format(self._name, index))
+        return np.load('{}_{}.npy'.format(self._name, index),
+                       allow_pickle=False)
 
     def get_file_list(self, datum_kwarg_gen):
         "This method is optional. It is not needed for access, but for export."
